project/SparkBuild.scala

```diff
@@ -476,6 +476,7 @@ object SparkParallelTestGrouping {
     "org.apache.spark.ml.classification.LogisticRegressionSuite",
     "org.apache.spark.ml.classification.LinearSVCSuite",
     "org.apache.spark.sql.SQLQueryTestSuite",
+    "org.apache.spark.sql.hive.client.HadoopVersionInfoSuite",
```

Comment:
AFAIK this only works for SBT. Maybe we need to disable the test by default to avoid breaking the maven build.

cc @gengliangwang

Reply:
This is fine. The test is not failing in maven build. In the maven build, it will still pass but won't be able to catch the issue.

Reply:
ah i see